src/components/users/Accounts.jsx

```diff
@@ -45,12 +47,17 @@ export function getAccountsState (
     };
 }
 
+export const cryptUserPassword = async (password) => {
+    const crypted = await python.spawn(encryptUserPw, password, { err: "message", environ: ["LC_ALL=C.UTF-8"] });
```

Comment:
Actually, don't you want to handle the error in the front end? uncaught exceptions will result in critical error and finish the installation. I guess we are fine with that, but also leaving the password non crypted is not the end of the world. An notification should be enough.

Reply:
Yes, it is on purpose. I think in such area (security) we should try to adhere to our policy/decision as much as possible. Also a notification would look a bit suspicious and puzzling to me as a user (like: hm, why are you crypting the password in the first place then, and what do I risk if I the password is not ecnrypted...). Also in the old UI we would fail hard with runtime error as well.